### PR TITLE
Update date Community Page

### PR DESCRIPTION
**Pull request by lee920217**
- Add content to `Community Page`
- Change `PostItem` style
- fixed `Layout` component and `Home` Page Error

### CODE REVIEW

File: src/components/PostItems/index.tsx [hunk 1]
```diff
@@ -42,10 +53,17 @@ const PostItems: React.FC<PostItemsProps> = ({
   showLastReplyToEvent = true,
   showFromCommunity = true,
   extraMenu,
-  extraHeader
+  extraHeader,
+  communityId= ""
```
Comment (digi-monkey) [inline src/components/PostItems/index.tsx:57]: why do we need this?

File: src/components/PostItems/PostCommunityHeader/index.tsx [hunk 2]
```diff
@@ -11,9 +11,11 @@ export function PostCommunityHeader({ event }: { event: Event }) {
   }
 
   return (
-    <div className={styles.communityPostTitle}>
-      <Icon type="icon-explore" />
-      From
+    <div className={styles.communityIdWrapper}>
+        From
+        <svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 18 18" fill="none">
```
Comment (digi-monkey) [inline src/components/PostItems/PostCommunityHeader/index.tsx:16]: let's keep Icon in the previous style using <Icon> component instead of raw svg

**Review by digi-monkey — CHANGES_REQUESTED** [review]

File: src/components/PostItems/index.tsx (hunk 1 above)
Comment (lee920217) [reply src/components/PostItems/index.tsx:57]: Initially, I wanted to design the 'create post' button within the 'postItem,' but later, it was moved to 'community,' and now it has been removed.

File: src/components/PostItems/index.tsx [hunk 3]
```diff
@@ -42,10 +53,17 @@ const PostItems: React.FC<PostItemsProps> = ({
...
 }) => {
+  const [activeTab, setActiveTab] = useState('Latest');
```
[the remark this answers is not in the document]
Comment (lee920217) [reply src/components/PostItems/index.tsx:59]: Initially, I wanted to design the 'create post' button within the 'postItem,' but later, it was moved to 'community,' and now it has been removed.

File: src/components/PostItems/index.tsx [hunk 4]
```diff
@@ -31,6 +41,7 @@ interface PostItemsProps {
     onClick: (event: Event, msg: typeof message) => any;
   }[];
   extraHeader?: React.ReactNode;
+  communityId?: string
```
Comment (digi-monkey) [inline src/components/PostItems/index.tsx:44]: please remove this

File: src/pages/communities/n/community.tsx [hunk 5]
```diff
@@ -408,51 +417,131 @@ export function Community({
...
+                    </div>
+                    {msgList.length === 0 && <Empty />}
+                    <PostItems
+                        msgList={msgList}
+                        worker={worker!}
+                        userMap={userMap}
+                        eventMap={eventMap}
+                        relays={worker?.relays.map(r => r.url) || []}
+                        showFromCommunity={false}
+                        communityId={community.id}
```
Comment (digi-monkey) [inline src/pages/communities/n/community.tsx:504]: please remove this

File: src/pages/communities/n/community.tsx [hunk 6]
```diff
@@ -408,51 +417,131 @@ export function Community({
...
+                              ? [
+                                {
+                                  label: 'approve this event',
+                                  onClick: (event, message) =>
+                                      createApproval(event, message),
+                                },
+                              ]
+                              : []
+                        }
+                        communityId={community.id}
```
Comment (digi-monkey) [inline src/pages/communities/n/community.tsx:537]: and here

**Review by digi-monkey — CHANGES_REQUESTED** [review]

**Comment by digi-monkey** [comment]
replace by 
- https://github.com/digi-monkey/flycat-web/pull/203

**Pull request closed without merging**